opus/application/static_media/js/detail.js

```diff
@@ -128,6 +128,23 @@ var o_detail = {
         o_detail.adjustDetailHeight();
     },
 
+    showDetailThumbInNav: function(imageHtml) {
+        if (imageHtml === undefined) {
+            if (opus.prefs.detail != "") {
+                let url = "/opus/api/image/thumb/" + opus.prefs.detail + ".json";
```

Comment:
This should use the internal API URL "__api/image/thumb/" so that we don't count these as uses by an external API user. We haven't needed this before so you'll have to add it to urls.py by just copying the normal "api/image" line and adding a new one with the "__" in front. Everything else can stay the same. The internal functionality doesn't change - we just use it when parsing the Apache logs.